lib/src/validation-server/spots/health.ts

```diff
@@ -0,0 +1,13 @@
+import { endpoint, request, response } from "../../lib";
+
+@endpoint({
+  method: "POST",
+  path: "/health"
+})
+export class HealthCheck {
+  @request
+  request() {}
+
+  @response({ status: 200 })
```

Comment:
What's the status when it's not healthy?

Reply:
This is just to check if the server was initialized or not, so if the server is not available it will return 503 I guess